tests/hmm/hmumu_utils.py

```diff
@@ -758,13 +771,13 @@ def finalize_weights(weights, all_weight_names=None):
 
     #multitply up all the nominal weights
     for this_syst in all_weight_names:
-        if this_syst == "nominal" or this_syst == "LHEScaleWeight" or this_syst == "mu1_id" or this_syst == "mu1_iso" or this_syst == "mu2_id"or this_syst == "mu2_iso":
+        if this_syst == "nominal" or this_syst == "LHEScaleWeight" or this_syst == "mu1_id" or this_syst == "mu1_iso" or this_syst == "mu2_id"or this_syst == "mu2_iso" or this_syst == "btag_weight_bFl" or this_syst == "btag_weight_cFl" or this_syst == "btag_weight_lFl":
```

Comment:
We should definitiely simplify this part of the code in the weight calculation... I can take care of that after you are done with this PR - we should aim to reduce the amount of manually written long sequences like that, as it will be hard to keep them all correct.